spec.go

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firebolt
+
+import (
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+// Specification returns the Plugin's Specification.
+func Specification() sdk.Specification {
+	return sdk.Specification{
+		Name:    "firebolt",
+		Summary: "The Firebolt plugin for Conduit, written in Go.",
+		Description: "The Firebolt connector is one of Conduit plugins." +
+			"It provides the source and destination firebolt connector.",
+		Version: "v0.1.0",
+		Author:  "Meroxa, Inc.",
```

Comment:
Feel free to add Yalantis here too (in this connector and others).

Reply:
thank!